cms/auth/tests/test_auth_integration.py

```diff
@@ -0,0 +1,341 @@
+import importlib
+import uuid
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase, override_settings
+from django.urls import reverse
+from wagtail import hooks
+
+from cms.auth import wagtail_hooks
+from cms.auth.tests.helpers import CognitoTokenMixin, build_jwt
+
+User = get_user_model()
```

Comment:
nitpick/non-blocking: we're importing from our users app directly in other places. `from cms.users.models import User`.

This pattern here is usually used in packages where you do not know what the user model is.

Reply:
Bad Sanjeev. I knew this as well. Updated in all the relevant places.